### PR TITLE
Fixing TypeHints

### PR DESCRIPTION
**Pull request by anishacharya**
type-hints should be int for the epochs ?

### CODE REVIEW

**Comment by philippmwirth** [comment]
Thanks for the pull-request @anishacharya, please remove the scaling of the return value then we can merge this 🙂

File: lightly/utils/scheduler.py [hunk 1]
```diff
@@ -75,9 +78,9 @@ class CosineWarmupScheduler(torch.optim.lr_scheduler.LambdaLR):
     def __init__(
         self,
         optimizer: torch.optim.Optimizer,
-        warmup_epochs: float,
-        max_epochs: float,
-        last_epoch: float = -1,
+        warmup_epochs: int,
+        max_epochs: int,
+        last_epoch: int = -1,
```
[the remark this answers is not in the document]
Comment (guarin) [reply lightly/utils/scheduler.py:83]: I had a look in more detail and I believe it makes sense to change the epoch arguments to `int`. The Pytorch schedulers expect an `int` for epoch values as can be seen in the [docs](https://pytorch.org/docs/stable/generated/torch.optim.lr_scheduler.LambdaLR.html#lambdalr) although it is not strictly typed in the code.

I also added a note about the naming to make it clear that the epoch arguments do not have to be actual epochs.

**Comment by guarin** [comment]
Updated the PR with the following changes:
* Type epoch and step arguments/parameters as int
* Convert warmup/max steps in benchmarks to int
* Add note about epoch naming convention

**Review by philippmwirth — APPROVED** [review]
